[2023-05-30T13:27:08.875+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: birthday_email_dag.send_birthday_emails manual__2023-05-30T13:11:52.972855+00:00 [queued]>
[2023-05-30T13:27:08.894+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: birthday_email_dag.send_birthday_emails manual__2023-05-30T13:11:52.972855+00:00 [queued]>
[2023-05-30T13:27:08.895+0000] {taskinstance.py:1308} INFO - Starting attempt 3 of 3
[2023-05-30T13:27:08.944+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): send_birthday_emails> on 2023-05-30 13:11:52.972855+00:00
[2023-05-30T13:27:08.950+0000] {standard_task_runner.py:57} INFO - Started process 605 to run task
[2023-05-30T13:27:08.956+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'birthday_email_dag', 'send_birthday_emails', 'manual__2023-05-30T13:11:52.972855+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/birthday_dag.py', '--cfg-path', '/tmp/tmp5qfnwxhw']
[2023-05-30T13:27:08.959+0000] {standard_task_runner.py:85} INFO - Job 179: Subtask send_birthday_emails
[2023-05-30T13:27:09.047+0000] {task_command.py:410} INFO - Running <TaskInstance: birthday_email_dag.send_birthday_emails manual__2023-05-30T13:11:52.972855+00:00 [running]> on host 014b75236497
[2023-05-30T13:27:09.217+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='birthday_email_dag' AIRFLOW_CTX_TASK_ID='send_birthday_emails' AIRFLOW_CTX_EXECUTION_DATE='2023-05-30T13:11:52.972855+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-30T13:11:52.972855+00:00'
[2023-05-30T13:27:09.219+0000] {sql.py:262} INFO - Executing: SELECT employee_email, employee_name FROM employee WHERE DATE_FORMAT(employee_date_of_birth, '%m-%d') = DATE_FORMAT(NOW(), '%m-%d')
[2023-05-30T13:27:09.234+0000] {base.py:73} INFO - Using connection ID 'hrms' for task execution.
[2023-05-30T13:27:09.375+0000] {base.py:73} INFO - Using connection ID 'hrms' for task execution.
[2023-05-30T13:27:09.380+0000] {sql.py:375} INFO - Running statement: SELECT employee_email, employee_name FROM employee WHERE DATE_FORMAT(employee_date_of_birth, '%m-%d') = DATE_FORMAT(NOW(), '%m-%d'), parameters: None
[2023-05-30T13:27:09.381+0000] {sql.py:384} INFO - Rows affected: 2
[2023-05-30T13:27:09.382+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/birthday_dag.py", line 30, in send_birthday_emails
    with open('/home/dsaidthimmareddy/Desktop/airflow_hrms/dags/Birthday_template.html', 'r') as template_file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/dsaidthimmareddy/Desktop/airflow_hrms/dags/Birthday_template.html'
[2023-05-30T13:27:09.394+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=birthday_email_dag, task_id=send_birthday_emails, execution_date=20230530T131152, start_date=20230530T132708, end_date=20230530T132709
[2023-05-30T13:27:09.424+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 179 for task send_birthday_emails ([Errno 2] No such file or directory: '/home/dsaidthimmareddy/Desktop/airflow_hrms/dags/Birthday_template.html'; 605)
[2023-05-30T13:27:09.456+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-05-30T13:27:09.487+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
